# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/tzdata/tzcode-makefile.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Fix tzcode's Makefile to actually be able the source code.

  - NoTag <notag@t2sde.org>

--- tzcode2024a/Makefile.vanilla	2024-03-20 16:55:54.759877057 +0100
+++ tzcode2024a/Makefile	2024-03-20 16:58:02.495591928 +0100
@@ -614,20 +614,12 @@
 # This list is not the same as the output of 'git ls-files', since
 # .gitignore is not distributed.
 VERSION_DEPS= \
-		calendars CONTRIBUTING LICENSE Makefile NEWS README SECURITY \
-		africa antarctica asctime.c asia australasia \
-		backward backzone \
-		checklinks.awk checknow.awk checktab.awk \
-		date.1 date.c difftime.c \
-		etcetera europe factory iso3166.tab \
-		leap-seconds.list leapseconds.awk localtime.c \
-		newctime.3 newstrftime.3 newtzset.3 northamerica \
-		private.h southamerica strftime.c theory.html \
-		time2posix.3 tz-art.html tz-how-to.html tz-link.html \
-		tzfile.5 tzfile.h tzselect.8 tzselect.ksh \
-		workman.sh zdump.8 zdump.c zic.8 zic.c \
-		ziguard.awk zishrink.awk \
-		zone.tab zone1970.tab zonenow.tab
+		asctime.c \
+		date.c difftime.c \
+		localtime.c \
+		private.h strftime.c \
+		tzfile.h \
+		zdump.c zic.c
 
 all:		tzselect zic zdump libtz.a $(TABDATA) \
 		  vanguard.zi main.zi rearguard.zi
--- tzcode2024a/Makefile.vanilla	2024-03-20 17:04:17.924703350 +0100
+++ tzcode2024a/Makefile	2024-03-20 17:05:41.890182825 +0100
@@ -621,8 +621,7 @@
 		tzfile.h \
 		zdump.c zic.c
 
-all:		tzselect zic zdump libtz.a $(TABDATA) \
-		  vanguard.zi main.zi rearguard.zi
+all:		tzselect zic zdump libtz.a
 
 ALL:		all date $(ENCHILADA)
 
--- tzcode2024a/Makefile.vanilla	2024-03-20 17:07:12.070696755 +0100
+++ tzcode2024a/Makefile	2024-03-20 17:10:18.127756466 +0100
@@ -625,25 +625,11 @@
 
 ALL:		all date $(ENCHILADA)
 
-install:	all $(DATA) $(REDO) $(MANS)
-		mkdir -p '$(DESTDIR)$(BINDIR)' \
-			'$(DESTDIR)$(ZDUMPDIR)' '$(DESTDIR)$(ZICDIR)' \
-			'$(DESTDIR)$(LIBDIR)' \
-			'$(DESTDIR)$(MANDIR)/man3' '$(DESTDIR)$(MANDIR)/man5' \
-			'$(DESTDIR)$(MANDIR)/man8'
-		$(ZIC_INSTALL) -l $(LOCALTIME) \
-			`case '$(POSIXRULES)' in ?*) echo '-p';; esac \
-			` $(POSIXRULES) \
-			-t '$(DESTDIR)$(TZDEFAULT)'
-		cp -f $(TABDATA) '$(DESTDIR)$(TZDIR)/.'
+install:	all
 		cp tzselect '$(DESTDIR)$(BINDIR)/.'
 		cp zdump '$(DESTDIR)$(ZDUMPDIR)/.'
 		cp zic '$(DESTDIR)$(ZICDIR)/.'
 		cp libtz.a '$(DESTDIR)$(LIBDIR)/.'
-		$(RANLIB) '$(DESTDIR)$(LIBDIR)/libtz.a'
-		cp -f newctime.3 newtzset.3 '$(DESTDIR)$(MANDIR)/man3/.'
-		cp -f tzfile.5 '$(DESTDIR)$(MANDIR)/man5/.'
-		cp -f tzselect.8 zdump.8 zic.8 '$(DESTDIR)$(MANDIR)/man8/.'
 
 INSTALL:	ALL install date.1
 		mkdir -p '$(DESTDIR)$(BINDIR)' '$(DESTDIR)$(MANDIR)/man1'
